code/datums/diseases/_disease.dm

```diff
@@ -239,7 +239,7 @@
 
 	. = cures.len
 	for(var/C_id in cures)
-		if(!affected_mob.reagents.has_reagent(C_id))
+		if(!affected_mob.reagents.has_reagent(target_reagent = C_id, check_subtypes = TRUE))
```

Comment:
Doesn't this affect other instances of subtype chems too?

One example is Nebula Nausea, cured by space cleaner, but this should make EZ Clean and Sterilizine work too. Dunno if you intended for it.

Reply:
Yeah, it will include subtypes in general. I gave it thought didn't see any cures where it'd be a big deal. to allow subtypes. Though could modify anxiety's cure proc if someone was strongly against including subtypes in general.

Reply:
Doesn't seem to be affecting anything except that, so should be good?